lang/python/flup/Makefile

```diff
@@ -9,39 +9,33 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=flup
 PKG_VERSION:=1.0.3
-PKG_RELEASE:=2
-PKG_LICENSE:=BSD-3-Clause
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/f/flup
 PKG_HASH:=5eb09f26eb0751f8380d8ac43d1dfb20e1d42eca0fa45ea9289fa532a79cd159
+
 PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=PKG-INFO
 
 include $(INCLUDE_DIR)/package.mk
-include ../python-package.mk
+include ../python3-package.mk
 
 define Package/flup
-  SUBMENU:=Python
   SECTION:=lang
   CATEGORY:=Languages
+  SUBMENU:=Python
   TITLE:=Random assortment of WSGI servers
   URL:=https://www.saddi.com/software/flup/
-  DEPENDS:=+python
+  DEPENDS:=+python3-light +python3-logging
+  VARIANT:=python3
 endef
 
 define Package/flup/description
   Random assortment of WSGI servers
 endef
 
-define Build/Compile
-	$(call Build/Compile/PyMod,,install --prefix=/usr --root=$(PKG_INSTALL_DIR))
-endef
-
-define Package/flup/install
-	$(INSTALL_DIR) $(1)$(PYTHON_PKG_DIR)
-	$(CP) \
-	    $(PKG_INSTALL_DIR)$(PYTHON_PKG_DIR)/* \
-	    $(1)$(PYTHON_PKG_DIR)
-endef
-
+$(eval $(call Py3Package,flup))
```

Comment:
`flup` is required for seafile;
and right now, seafile & django stuff is dependent on python;
this also means converting django packages to  python-django-xxx & python3-django-xxx

so, for now, i would convert this to  `python-flup` & `python3-flup` + updating deps;
then we can remove `python-flup` and `python-xxx` 

because of seafile, jumping from python to python3 needs some synchronization of things [for which I am doing some preparation in chunks of PRs]

Reply:
Agree! I will do it.